libs/pika/async_mpi/include/pika/async_mpi/trigger_mpi.hpp

```diff
@@ -85,6 +85,8 @@ namespace pika::mpi::experimental::detail {
             bool completed{false};
             pika::detail::spinlock mutex;
             pika::condition_variable cond_var;
+            // MPIX
```

Comment:
General question: should we refer to this as "MPI continuations" rather than "MPIX"? If I understand correctly, MPIX generally refers to MPI "extensions", right? By this I include variable names/typedefs etc. as well.

Reply:
I don't really mind what we call it, they are mpi continuations in the mpi extensions header, so I thought mpix_continuations was a good name. in the header they use `OMPI_HAVE_MPI_EXT_CONTINUE` so mpi_ext_continuations? The functions are `MPIX_Continue_cb_function` etc, so MPIX seemed like a goood prefix for me to use for now. I will replace some commmenrtrs and debug stuff with `MPI_EXT_CONTINUE`